src/ClientData/include/ClientData/DataManager.h

```diff
@@ -127,18 +128,18 @@ class DataManager final {
 
  private:
   const std::thread::id main_thread_id_;
-  orbit_client_data::FunctionInfoSet selected_functions_;
-  absl::flat_hash_set<uint64_t> visible_function_ids_;
+  FunctionInfoSet selected_functions_;
+  absl::lts_2020_09_23::flat_hash_set<uint64_t> visible_function_ids_;
```

Comment:
Is this intended? Could you explain me what are the advantages of this?

Reply:
ops... not intended.